tests/no_error_logs_test.py

```
@@ -0,0 +1,91 @@
+        error_logs = [log for log in logs if log["type"] == "error"]
+
+        # Detailed assertion message for easier debugging
+        error_details = "\n".join(
+            [f"[{err['type']}] {err['text']}" for err in error_logs]
+        )
+        assert (
+            len(error_logs) == 0
+        ), f"Found {len(error_logs)} error(s) in console:\n{error_details}"
+
+    except Exception as e:
+        print(f"Test: 'test_no_error_logs_after_load' failed: {str(e)}")
```

Ruff (0.15.2)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)